README.md

```diff
@@ -1 +1,97 @@
-# TDEI-python-gtfs-pathways-validation
\ No newline at end of file
+# TDEI-python-gtfs-pathways-validation
+
+## Introduction 
+Service to Validate the GTFS pathways file that is uploaded. At the moment, the service does the following:
+- Listens to the topic _gtfs-pathways-validation_ for any new message (that is triggered when a file is uploaded)
+- Consumes the message and checks the filename in the message
+  - if the filename contains the word _valid_ returns **valid** as the result
+  - if the filename contains the word _invalid_ returns **invalid** as the result
+  - if the file name does not contain either, it will return **invalid** as the result
+- Publishes the result to the topic _gtfs-pathways-validation_
+
+## Getting Started
+The project is built on Python with FastAPI framework. All the regular nuances for a Python project are valid for this.
+
+### System requirements
+| Software   | Version |
+|------------|---------|
+| Python     | 3.10.x  |
+
+
+### Connectivity to cloud
+- Connecting this to cloud will need the following in the `.env` file
+
+```bash
+UPLOAD_TOPIC=xxxx
+UPLOAD_SUBSCRIPTION=xxxx
+VALIDATION_TOPIC=xxxx
+QUEUECONNECTION=xxxx
+STORAGECONNECTION=xxxx
+```
+The application right now does not connect with the storage but validates via the file name.
+
+### How to Setup and Build
+Follow the steps to install the node packages required for both building and running the application
+
+1. Setup virtual environment
+    ```
+    python3.10 -m venv .venv
+    source .venv/bin/activate
+    ```
+
+2. Install the dependencies. Run the following command in terminal on the same directory as `requirements.txt`
+    ```
+    # Installing requirements
+    pip install -r requirements.txt
+    # Installing tdei-gtfs-csv-validator package
+    pip install -i https://test.pypi.org/simple/ tdei-gtfs-csv-validator
```

Comment:
this line is not working. change to 
` pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple tdei-gtfs-csv-validator==0.0.28`

Reply:
this is not needed, it you run `pip install -i https://test.pypi.org/simple/ tdei-gtfs-csv-validator` by default it will install latest package

Reply:
we dont want the latest always. We need to know exactly which version we are using. so please add ==0.0.28